test/controllers/BouncedEmailControllerSpec.scala

```diff
@@ -31,30 +31,44 @@ class BouncedEmailControllerSpec extends TestUtil {
 
   "The .process action" when {
 
-    "a valid JSON body is received" should {
+    "allowEventHubRequest feature switch is on" should {
 
-      "return 200" in {
-        val request = FakeRequest("POST", "/", Headers((CONTENT_TYPE, JSON)), bouncedEmailMaxJson)
-        val result = controller.process(request)
-        status(result) shouldBe Status.OK
+      "a valid JSON body is received" should {
```

Comment:
I think the two "shoulds" should be in one e.g. "the feature switch is on and a valid json body is received"